CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 ### Enhancements
 * Features added by release v6.0.26:
   * Added ability to replace last sort descriptor on DescriptorOrdering in addition to append/prepending to it [#3884]
+* Added Apple Silicon build
```

Comment:
As much as I like to see Apple Silicon here, we should probably say "Add arm64 to OSX build script" or something similar.

Also note that (even though the CHANGELOG doesn't seem to actively follow this) CHANGELOG entries should always be in the imperative; in other words, `Add arm64` not `Added arm64`.